Documentation/ceph-quickstart.md

```diff
@@ -53,11 +53,64 @@ make sure you set the `dataDirHostPath` property. For more settings, see the doc
 Save the cluster spec as `cluster.yaml`:
 
 ```yaml
+#######################################################
```

Comment:
i think this idea is fine to show some visual separation between the RBAC stuff at the top and the real Cluster CRD content below.  The content trailing outside the bounds of the comment block is bothering me though, maybe this block content would work better?
```
#################################################################################
# This example first defines some necessary namespace and RBAC security objects.
# The actual Ceph Cluster CRD example can be found at the bottom of this example.
#################################################################################
...
<omitted>
...
#################################################################################
# The Ceph Cluster CRD example
#################################################################################
```